style.css

```diff
@@ -0,0 +1,1381 @@
+.textcontainer {
+  display: flex;
+  flex-direction: column;
+}
+
+.imagecontainer {
+  display: flex;
+  justify-content: space-between;
+}
+
+.defineimage {
+  width: 111px;
+  height: 74px;
+  background-image: url(left_topside_icon.svg);
+}
+
+.defineimage1 {
+  width: 20px;
+  height: 14px;
+  position: relative;
+  top: 15px;
+  background-image: url(ic_menu.svg);
+}
+
+.secondimagecontain {
+  background-image: url("image_geometry_2-1.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
+  height: 180px;
+  top: 345px;
+  position: relative;
+}
+
+.border-button {
+  border: solid 1px #00f;
+  display: inline-block;
+  position: relative;
+  text-align: center;
+  left: 50%;
+  margin-left: -47.65%;
+  top: 310px;
+  width: 340px;
+  height: 50px;
+  line-height: 50px;
+  color: white;
+  text-decoration: none;
+  background: black;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+}
+
+.border-button:hover {
+  border-width: 5px;
+}
+
+.text1 {
+  position: absolute;
+  left: 16px;
+  top: 70px;
+  padding-left: 10px;
+  font-family: 'Playfair Display', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 50px;
+  line-height: 100%;
+  color: #1a2236;
+}
+
+.text2 {
+  position: absolute;
+  width: 320px;
+  height: 126px;
+  left: 16px;
+  top: 224px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 21px;
+  color: #1a2236;
+}
```

Comment:
You did nice work here with the positioning of all of these text elements. I commend you. It does not look easy to do.
However, you could just simply style them as different elements with padding and margin to affect this design.

It would save you a lot of time and code. 
For instance, you have until class text15.

The proper use of CSS for a DRY Do not Repeat Yourself design is just having one class do the styling for many different elements that are very similar.

I highlighted the text1 through text3 styles in your index.html.

This [article](https://www.freecodecamp.org/news/7-important-tips-for-writing-better-css/) on some CSS tricks for writing better CSS can really help you a lot and save you a lot of time

Reply:
No compulsory changes are suggested to be made here. The comment said "you could simply style them...". I will definitely incorporate this in my next project. Changing this now would take a lot of time and is not convenient with my present coding. I will work on it definitely. Thank you for your valuable advice.